test/test.sh

```diff
@@ -559,27 +559,223 @@ test_max_client_conn() {
 
 # - max pool size
 test_pool_size() {
+	rm -f $LOGDIR/test.tmp
+
 	# make existing connections go away
 	psql -X -p $PG_PORT -d postgres -c "select pg_terminate_backend(pid) from pg_stat_activity where usename='bouncer'"
 	until test $(psql -X -p $PG_PORT -d postgres -tAq -c "select count(1) from pg_stat_activity where usename='bouncer'") -eq 0; do sleep 0.1; done
 
-	docount() {
```

Comment:
We split up `docount` method to spawn and count connections for  for user/database seperately.